study/cyber-tutor-2/js/app.js

```diff
@@ -0,0 +1,204 @@
+let myQuestions = [
+  {
+    question: "A hacker created malware that looks like an adblocker but actually takes over the user's computer. What is this malware called?",
+    correctAnswer: "Trojan",
+	responseVal: false
+  },
+  {
+    question: "Keylogging occurs when a program captures what from a victim?",
+    correctAnswer: "Keystrokes",
+	responseVal: false
+  },
+  {
+    question: "Bella receives an email from her insurance company asking her to send them her account information. Little does she know, a hacker is posing as the company so he can get her data. This kind of attack is called: ",
+    correctAnswer: "Phishing",
+	responseVal: false
+  },
+  {
+    question: "Jacob accidentally installs a malware. But, he updated his software and fixed a vulnerability before the malware was activated, so his computer was not taken over. The process of installing system updates is called: ",
+    correctAnswer: "Patching",
+	responseVal: false
+  },
+  {
+    question: "A hacker realizes that Julie's system has a vulnerability; he takes advantage of it using a program. What is this program called?",
+    correctAnswer: "Exploit",
+	responseVal: false
+  },
+  {
+    question: "Henry goes on nvd.nist.gov. What does he find?",
+    correctAnswer: "CVEs",
+	responseVal: false
+  },
+];
+
+/*const ALL_PROBLEM_TYPES = [
+    "forLoop",
+    "nestedForLoop",
+    "whileLoop"
+];*/
+
+// globals
+let score = 0;
+//let selectedProblemTypes = JSON.parse(JSON.stringify(ALL_PROBLEM_TYPES));
+let problem, answer;
+let number = 0;
+// initialize UI components
+
+showProblem();
+
+setAnswer();
+//createProblemTypeCheckboxes();
+wordBank();
+// bind onclick functions to the buttons
+document.getElementById("submit").onclick = checkAnswer;
+
+function showProblem(){
+	problem = myQuestions[number].question;
+ // show problem text to user
+    updatePrompt(problem);
+
+}
+function setAnswer(){
+	answer = myQuestions[number].correctAnswer;
+}
+
+
+/*function createProblemTypeCheckboxes() {
```

Comment:
You most likely won't ever need most of the commented code, so you can just delete it.